app/views/groups/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<header class="home">
+<div class="toolbar">
+  <%= link_to image_tag("https://img.icons8.com/officel/344/back.png"),groups_path, class: "back_btn" %>
+  <h5 class="toolbar__groups">Groups</h5>
+</div>
+</header>
+
+<% if flash[:notice] %>
+  <div class="flash-notice">
+    <p><%= notice %></p>
+  </div>
+<% end %>
+
+<section class="main-container">
+  <div id="groups">
+    <% @groups.each do |group| %>
+      <%= render group %>
+    <% end %>
+  </div>
+  <div class="splash-buttons">
+    <%= link_to "New group", new_group_path %>
+  </div>
+</section>
```

Comment:
- [ ] [REQUIRED] Hello @AbuTalha3 , You actually have a wonderful application here, and I commend your effort for this 🥇.
However, I noticed a potential enhancement for a better user experience. It would be beneficial if, while on the category/group page, users could navigate to the transaction page by simply clicking on any of the listed categories/groups. Currently, the system automatically directs users to the transaction page when adding a new category/group, which is a good feature. Nevertheless, it is essential to include the capability for users to access the transaction page by clicking on any existing categories/groups directly from the category/group page. This enhancement would contribute significantly to the overall usability of your application.

### Take a look at the project requirement
<img width="860" alt="Screenshot 2024-01-15 at 09 59 15" src="https://github.com/AbuTalha3/budget-app/assets/58692590/4dd79783-e246-436d-8934-5428d29336a0">

### Make the categories of your design clickable

<img width="416" alt="Screenshot 2024-01-15 at 09 41 02" src="https://github.com/AbuTalha3/budget-app/assets/58692590/b16fb590-cb3c-4dc2-95e4-eab17726a77c">